Project-Service-implementation-Script/main.py

```diff
@@ -146,9 +156,8 @@ def programCreation(accessToken,parentFolder,externalId,pName,pDescription,roles
       "components" : [
       ],
        "scope": {
-            "entityType": scopeEntityType,
-            "entities": entitiesPGMID,
-            "roles": roles
+            entitiesTypeStr: entitiesPGMID,
```

Comment:
@MallanagoudaB why we are changing scope for adding metInforamtion @Prajwal17Tunerlabs

Reply:
This change was supposed be done very long ago @VISHNUDAS-tunerlabs . But we somehow missed it. You remember we made changes to `solution scope structure` few months ago? Same was to be done for program scope structure. While testing on Wednesday we found this issue and fixed under this PR itself.